rackcity/views/user_views.py

```diff
@@ -60,7 +60,7 @@ def netid_login(request):
         user.save()
     token, _ = Token.objects.get_or_create(user=user)
     return JsonResponse(
-        {"token": token.key},
+        {"key": token.key},
```

Comment:
rename to match response from `rest-auth/login/`

Reply:
good .